hooks/useContractorInterest.ts

```
@@ -0,0 +1,189 @@
+"use client";
+
+import { useState, useEffect, useCallback } from "react";
+import { InterestedContractorsService } from "@/app/lib/services/interestedContractors";
+import type {
+  InterestedContractor,
+  ContractorFilters,
+  ContractorListResponse,
```

ESLint

[error] 'ContractorListResponse' is defined but never used.

(@typescript-eslint/no-unused-vars)